site-src/geps/pending-geps.md

```diff
@@ -0,0 +1,58 @@
+| **Emissary**        |                               |                                      |                                    |                     |                                       |                                |                               |                                         |                                |                              |                                                               |                            |                                      |                                                     |                                       |
+| **Envoy Gateway**   |                               |                                      |                                    |                     |                                       |                                |                               |                                         |                                |                              |                                                               |                            |                                      |                                                     |                                       |
+| **Flomesh**         |                               |                                      |                                    |                     |                                       |                                |                               |                                         |                                |                              |                                                               |                            |                                      |                                                     |                                       |
+| **Gloo**            |                               |                                      |                                    |                     |                                       |                                |                               |                                         |                                |                              |                                                               |                            |                                      |                                                     |                                       |
+| **GKE**             |                               |                                      |                                    |                     |                                       |                                |                               |                                         |                                |                              |                                                               |                            |                                      |                                                     |                                       |
```

Comment:
GKE currently supports "GEP-1748: Gateway API Interaction with Multi-Cluster Services", the rest are all "not planned" at this point as we only support the standard channel.

Reply:
I'll make the change.

Reply:
But GEP-1748 is still listed as experimental.